supportedBackends/htcondor/src/main/scala/cromwell/backend/impl/htcondor/HtCondorJobExecutionActor.scala

```diff
@@ -79,7 +106,16 @@ class HtCondorJobExecutionActor(override val jobDescriptor: BackendJobDescriptor
   /**
     * Execute a new job.
     */
-  override def execute: Future[BackendJobExecutionResponse] = Future(executeTask())
+  override def execute: Future[BackendJobExecutionResponse] = {
+    log.debug("{} Checking if hash {{}} is in the cache.", tag, jobHash)
```

Comment:
Here, it should be a pattern match or if /else with the enableCaching flag, if enabled do this whole sequence of steps, if not then simply execute 
```scala
createExecutionFolderAndScript
executionResponse success executeTask()

```